docs/source/enable/traitsui.rst

```diff
@@ -0,0 +1,27 @@
+Enable TraitsUI Editors
+=======================
+To facilitate the inclusion of Enable :class:`~.Component` objects in
+`TraitsUI GUIs <https://docs.enthought.com/traitsui>`_, Enable provides
+:class:`~.ComponentEditor`.
```

Comment:
The docstring of `ComponentEditor` says "wxPython editor factory for Enable components."
and this is all you effectively see at this link.

It would be useful to change this docstring and add #: for the few traits we expect to be used

Reply:
Good catch. I'm not sure where you're suggesting I put the "#: for the few traits we expect to be used" comment.

Reply:
For example on this line:
https://github.com/enthought/enable/blob/e40a8a952376da67fd5c4b0fb9856c3a093f49a2/enable/component_editor.py#L102
So that the `high_resolution` trait and this comment show up in the api docs

Reply:
Ahhhh. OK.